### PR TITLE
add rct1 "scrambled eggs" preview colour

### PR DESCRIPTION
**Pull request by 733737**
adds the rct1 colour preset scheme to the ride presets
Twist_RCT1_Icon

### CODE REVIEW

File: objects/rct2/ride/rct2.ride.twist1.json [hunk 1]
```diff
@@ -24,6 +24,13 @@
                     "yellow",
                     "black"
                 ]
+            ],
+		    [
```
Comment (ZeeMaji) [inline objects/rct2/ride/rct2.ride.twist1.json:28]: This extra bracket should be removed, i don't think the object will load correctly with this present.

Comment (733737) [reply objects/rct2/ride/rct2.ride.twist1.json:28]: the way it is now follows the convention of the object. Removing it isn't logical as far as I know.

Comment (Gymnasiast) [reply objects/rct2/ride/rct2.ride.twist1.json:28]: Looks like it is correct, it’s just that the indentation is off.

Comment (ZeeMaji) [reply objects/rct2/ride/rct2.ride.twist1.json:28]: My bad, i guess it is just the indentation.

Comment (733737) [reply objects/rct2/ride/rct2.ride.twist1.json:28]: it appears correct in notepad++, which is what I used to edit it. I suppose I could edit the indentation in the browser.

**Comment by 733737** [comment]
I haven't looked into it. I was watching twitch the other day and someone was streaming rct deluxe and I noticed the twist thumbnail. I could simply google all of the rct1 thumbnails from the wiki and compare them to the ingame presets however.

**Comment by Gymnasiast** [comment]
It looks like you mixed tabs with spaces (even after the latest commit). Please use spaces throughout.

**Comment by 733737** [comment]
should be fixed

**Pull request merged**